modules/ROOT/pages/runtime-monitoring.adoc

```diff
@@ -232,13 +120,65 @@ Sample extract of exposed Prometheus data:
  # TYPE tomcat_servlet_request_max_seconds gauge
  tomcat_servlet_request_max_seconds{name="default",} 0.0
  tomcat_servlet_request_max_seconds{name="dispatcherServlet",} 0.104
- # HELP tomcat_threads_config_max_threads
- # TYPE tomcat_threads_config_max_threads gauge
- tomcat_threads_config_max_threads{name="http-nio-8080",} 200.0
- # HELP tomcat_sessions_expired_sessions_total
- # TYPE tomcat_sessions_expired_sessions_total counter
- tomcat_sessions_expired_sessions_total 0.0
- # HELP tomcat_sessions_active_max_sessions
- # TYPE tomcat_sessions_active_max_sessions gauge
- tomcat_sessions_active_max_sessions 0.0
  ...
+
+
+== Configuration
+
+
+Retrieve xref:BonitaBPM_platform_setup.adoc]#update_platform_conf[current configuration] by running:
+
+[source,bash]
+----
+./setup/setup.sh pull
+----
+
+Edit file `./setup/platform_conf/current/platform_engine/bonita-platform-sp-custom.properties` +
+
+You will see, in the `# MONITORING` section, a series of properties with their default value:
+
+In the first section, published can be activated and configured.
+
+ # PUBLISHERS = where to publish?
+ # publish metrics to Prometheus
+ com.bonitasoft.engine.plugin.monitoring.publisher.prometheus.enable=false
```

Comment:
This property the only one having `.plugin` in its name. Should we change it?

Reply:
I would say we don't change properties right here, because it's a breaking change.
If we do we should review conventions more globally: https://bonitasoft.atlassian.net/wiki/spaces/BONITA/pages/9678946496/Configuration+management